### ISSUE TITLE
API: Implement A1a into mappers

### ISSUE THREAD

**Issue by ashleybot** [UK-SBCoA/uniform-data-set-dotnet-api#23]
Depends on #21 and #22 

Write methods to map A1a entity to A1a Dto.
Write methods to map A1a Dto to A1a entity.

**Comment by ashleybot** [comment]
@mlan225 Reviewing another PR reminded me that when you finish the mapping for A1a that will be the time to updated the solution/project version numbers and tag the commit so the api client library nuget package gets automatically built.
